Added mod,divide,multiply

ijassiem opened the pull request:
Added code with modulus, division and multiplication functions.

ijassiem committed:
Added mod,divide,multiply

james-smith-za commented on divide.py:
```diff
@@ -0,0 +1,11 @@
+
+def divide(a, b):
+    """Divide two numbers."""
+    return a * b
```
Oops, this should be `/`.

ijassiem replied:
Fixed

james-smith-za commented on mod.py:
```diff
@@ -0,0 +1,11 @@
+
+
+def mod(a, b):
+    """Multiply two numbers."""
```
Docstring isn't correct.

ijassiem replied:
FIxed

james-smith-za commented on multiply.py:
```diff
@@ -0,0 +1,11 @@
+"""
+This is an example docstring.
+
+A docstring with a summary line, a blank line
+and some more text like here.
+"""
```
I'd suggest not just copy-pasting docstrings for the sake of having them.

It's a bit of an art to write good docstrings, and in general the more practice you have at deliberately / intentionally making good ones, the better, because you'll develop more of an intuition then as a result of trying more things.

tyronevb reviewed (commented):
I'm specifically looking at the commit message. I understand that this might not be the final commit, but since it's in a PR staged for review, I'd like to point out a few things.

1. Your commit message doesn't follow the recommended structure i.e. having an imperative mood summary line, followed by a description of what was done and why. A summary more in line with the proposed structure could be "Add capability to perform modulus, division and multiplication operations". Or something along those lines. Then in subsequent paragraphs of the commit message, you could explain what was added why.
2. A reference to the Jira ticket that concerns this piece of work isn't included.
3. I could nitpick, and I guess it does come down to style and preference, but in this context, each mathematical operation may be considered an individual feature, which would warrant separate commits/PRs. If they're being used to achieve something else i.e. some other function or feature that uses these operations, then the "final" commit message should summarise that feature. (If that makes sense)

Have a look at the documentation on version control, but I understand you might've done things differently on your other PR that I am yet to review ;)

ijassiem committed:
Implement changes relating to comments in pull request "Added mod, divide multiply #1"

Added descriptive docstrings and corrected typos in the divide.py, mod.py and multiply.py.

Resolves: "DGDT-11"
